applications/GeoMechanicsApplication/tests/cpp_tests/test_equation_of_motion.cpp

```diff
@@ -120,4 +119,39 @@ KRATOS_TEST_CASE_IN_SUITE(CalculateMassMatrix3D4NGivesCorrectResults, KratosGeoM
     KRATOS_CHECK_MATRIX_NEAR(mass_matrix, expected_mass_matrix, 1e-4)
 }
 
+KRATOS_TEST_CASE_IN_SUITE(CalculateDampingMatrixGivesCorrectResults, KratosGeoMechanicsFastSuite)
+{
+    constexpr std::size_t n = 10;
+
+    constexpr double mass_matrix_value = 10;
+    const auto       mass_matrix       = scalar_matrix(n, n, mass_matrix_value);
+
+    constexpr double stiffness_matrix_value = 20;
+    const auto       stiffness_matrix       = scalar_matrix(n, n, stiffness_matrix_value);
+
+    double rayleigh_alpha = 0.0;
+    double rayleigh_beta  = 1.0;
+    auto   damping_matrix = GeoEquationOfMotionUtilities::CalculateDampingMatrix(
+        rayleigh_alpha, rayleigh_beta, mass_matrix, stiffness_matrix);
+
+    KRATOS_CHECK_MATRIX_NEAR(damping_matrix, stiffness_matrix, 1e-4)
+
+    rayleigh_alpha = 1.0;
+    rayleigh_beta  = 0.0;
+    damping_matrix = GeoEquationOfMotionUtilities::CalculateDampingMatrix(
+        rayleigh_alpha, rayleigh_beta, mass_matrix, stiffness_matrix);
+
+    KRATOS_CHECK_MATRIX_NEAR(damping_matrix, mass_matrix, 1e-4)
+
+    rayleigh_alpha = 0.5;
+    rayleigh_beta  = 0.5;
+    damping_matrix = GeoEquationOfMotionUtilities::CalculateDampingMatrix(
+        rayleigh_alpha, rayleigh_beta, mass_matrix, stiffness_matrix);
+
+    const double expected_matrix_value = rayleigh_alpha * mass_matrix_value + rayleigh_beta * stiffness_matrix_value;
```

Comment:
Excellent unit test! The only thing I'm a bit hesitant about is repeating the implementation of the damping matrix computation here. Instead, we could also express the expected matrix like you did in the unit test above this one (see `expected_mass_matrix`).

Reply:
Since the expected matrix only has a single value, I would not express the matrix in the same way as `expected_mass_matrix`. We might want to give this double just the value it should have (which I guess is 15.0) and use `expected_damping_matrix = scalar_matrix(n, n, expected_matrix_value)` to fill the matrix as we do now. Not sure what you both think?

Reply:
Agreed with Richard. Now the calculated damping matrix is compared with expected damping matrix only. Each time the expected matrix is filled with a specific single value.